packages/s2-core/src/facet/table-facet.ts

```diff
@@ -404,6 +404,11 @@ export class TableFacet extends BaseFacet {
       opacity: style?.horizontalBorderColorOpacity,
     };
 
+    const shadowAttrs = {
+      shadowColor: style?.horizontalBorderColor,
```

Comment:
跟pivot 保持一致吧
```suggestion
       fill: `l (0) 0:${splitLine.shadowColors?.left} 1:${splitLine.shadowColors?.right}`,
```
但是需要区分下方向